routing/payment_lifecycle_test.go

```diff
@@ -0,0 +1,653 @@
+package routing
```

Comment:
Add to commit message "PURE MOVE"? (is that our standard?)

Reply:
Added "(almost) PURE CODE MOVE" to commit message.